pkg/pxc/users/users.go

```diff
@@ -0,0 +1,73 @@
+package users
+
+import (
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/pkg/errors"
+)
+
+type Manager struct {
+	db *sql.DB
+}
+
+type SysUser struct {
+	Name  string   `yaml:"username"`
+	Pass  string   `yaml:"password"`
+	Hosts []string `yaml:"hosts"`
+}
+
+func NewManager(hosts []string, rootPass string) (Manager, error) {
+	var um Manager
+	var err error
+	for _, host := range hosts {
+		mysqlDB, err := sql.Open("mysql", "root:"+rootPass+"@tcp("+host+")/?interpolateParams=true")
+		if err != nil {
+			continue
+		}
+		um.db = mysqlDB
+		break
+	}
+	if um.db == nil {
+		return um, errors.Wrap(err, "cannot connect to any host")
+	}
+
+	return um, nil
+}
+
+func (u *Manager) UpdateUsersPass(users []SysUser) error {
+	defer u.db.Close()
+	tx, err := u.db.Begin()
+	if err != nil {
+		return errors.Wrap(err, "begin transaction")
+	}
+
+	_, err = tx.Exec("FLUSH PRIVILEGES")
+	if err != nil {
+		tx.Rollback()
```

Comment:
I know it's strange, but just for me :)
If you ignore the returned errors could you please do it explicit way?
`_ = tx.Rollback()`
My linter goes crazy without `_ =`

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/418/commits/9a770955143b4e0e1b00e53653fdb45ab333357e